index.html

```diff
@@ -0,0 +1,538 @@
+                        </div>
+                    </div>
+                    <div class="box4">
+                        <a target="_blank" href="Batman.html"><img src="/images/notifications-blue.svg" alt="Twitter
+                        messages button"></a>
+                        <div class="disappear">
+                            <h2><a href="Batman.html">Messages</a></h2>
+                        </div>
+                    </div>
+                    <div class="box5">
+                        <a target="_blank" href="Batman.html"><img src="/images/messages-blue.svg" alt="Twitter
+                        bookmarks button"></a>
+                        <div class="disappear">
+                            <h2><a href="Batman.html">Bookmarks</a></h2>
+                        </div>
+                    </div>
+                    <div class="box6">
+                        <a target="_blank" href="Batman.html"><img src="/images/bookmarks-blue.svg" alt="Twitter
+                        lists button"></a>
+                        <div class="disappear">
+                            <h2><a href="Batman.html">Lists</a></h2>
+                        </div>
+                    </div>
+                    <div class="box7">
+                        <a target="_blank" href="Batman.html"><img src="/images/lists-blue.svg" alt="Twitter
+                        profile page button"></a>
+                        <div class="disappear">
+                            <h2><a href="Batman.html">Profile</a></h2>
+                        </div>
+                    </div>
+                    <div class="box8">
+                        <a target="_blank" href="Batman.html"><img src="/images/more-blue.svg" alt="Twitter
+                        more button"></a>
+                        <div class="disappear">
+                            <h2><a href="Batman.html">More</a></h2>
+                        </div>
+                    </div>
+
+                    <div button class="box9">
+                        <a href="Batman.html" class="lefttweet">Tweet</a>
+                    </div>
+
+
+
+
+
+                    <div class="box10">
+
+                        <div class="profile-pic">
+                            <a target="_blank" href="Batman.html"><img src="images/pic1.jpg" alt="Profile picture">
+                            </a>
+                        </div>
+
+                        <div class="deets">
+                            <h4>Mr. Barry</h4>
+                            <h3>@MrBarry</h3>
+                        </div>
+                    </div>
+
+                </div>
+
+            </div>
+
+        </div>
+
+
+        <!-- MIDDLE COLUMN -->
+
+
+        <!-- TOP BOX /MID COLUMN -->
+        <div class="main-col">
+            <div class="boxone">
+                <div class="homeone">
+                    <div class="top-home">Home</div>
+                    <div class="top-tweets">
+                        <svg viewBox="0 0 24 24">
```

Comment:
svg's should not be inside the html, but be in a seperate file for all your svg's. You should use an <img> tag and then point to the svg file with the src.